ngrinder-core/src/main/java/net/grinder/message/console/AgentControllerState.java

```diff
@@ -42,6 +42,7 @@
 	/**
 	 * When agent controller is down.
 	 */
+	// TODO: Remove INACTIVE. If agent state is shared by hazelcast, the agent not exist in shared state map is INACTIVE
```

Comment:
@imbyungjun 

https://github.com/naver/ngrinder/blob/feature/clean-up/ngrinder-controller/src/main/resources/ngrinder_datachange_logfile/db.changelog_schema_MYSQL.xml#L72

This field no longer necessary in mysql.
It may be make error during migration cubrid to mysql.